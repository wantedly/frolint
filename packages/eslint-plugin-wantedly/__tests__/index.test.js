const plugin = require("..");

it("should match snapshot", () => {
  expect(plugin).toMatchSnapshot();
});
