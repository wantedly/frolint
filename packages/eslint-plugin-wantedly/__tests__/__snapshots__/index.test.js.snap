// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "rules": Object {
    "graphql-operation-name": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/graphql-operation-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "graphql-pascal-case-type-name": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/graphql-pascal-case-type-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-camel-case-field-name": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-camel-case-field-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-pascal-case-type-name": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-pascal-case-type-name.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
    "nexus-upper-case-enum-members": Object {
      "create": [Function],
      "meta": Object {
        "docs": Object {
          "url": "https://github.com/wantedly/frolint/tree/master/packages/eslint-plugin-wantedly/docs/rules/nexus-upper-case-enum-members.md",
        },
        "fixable": "code",
        "type": "suggestion",
      },
    },
  },
}
`;
